tests/func/test_diff.py

```diff
@@ -226,3 +226,20 @@ def test(self):
             ],
         }
         self.assertEqual(test_dct, result)
+
+
+class TestDiffModifiedFileMessage(TestDiffModifiedFile):
```

Comment:
if there is not new logic to setup, shutdown, etc. let's make it a method in TestDiffModifiedFile, do not wrap it into a separate class

Reply:
Fixed